storage/innobase/log/log0recv.cc

```diff
@@ -4602,6 +4602,9 @@ dberr_t recv_recovery_from_checkpoint_start()
 					LSN_PF, recv_sys.lsn);
                         goto err_exit;
 		}
+		if (recv_sys.is_corrupt_fs()) {
+			goto err_exit;
+		}
```

Comment:
Here we are not holding `recv_sys.mutex`, but it should not really matter much, because this is the only thread that can modify any `recv_sys` members at this point.

Reply:
Makes sense. Modified the code.